### PR TITLE
SSCS-10222 Add new field inside CCD for hearing state

### PR DESCRIPTION
**Pull request by MohammedHMCTS**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SSCS-10222

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json [hunk 1]
```diff
@@ -676,6 +676,7 @@
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "exceptionRecordReference", "UserRole": "caseworker-sscs-clerk", "CRUD": "R"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "furtherEvidenceAction", "UserRole": "caseworker-sscs-clerk", "CRUD": "CRU"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "hearings", "UserRole": "caseworker-sscs-clerk", "CRUD": "CRUD"},
+  {"LiveFrom": "04/03/2022", "CaseTypeID": "Benefit", "CaseFieldID": "hearingRoute", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"},
```
Comment (dattipoeHMCTS) [inline benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json:679]: Should check what the permissions need to be for this

File: benefit/data/sheets/CaseField/CaseField.json [hunk 2]
```diff
@@ -419,6 +419,6 @@
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "ID": "caseAccessCategory", "Label": "Case access category", "FieldType": "Text", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "ID": "caseName", "Label": "Case name", "FieldType": "Text", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "ID": "caseManagementLocation", "Label": "Case management location", "FieldType": "CaseManagementLocation", "SecurityClassification": "PUBLIC"},
-  {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "ID": "SearchCriteria ", "Label": "Search Criteria", "FieldType": "SearchCriteria ", "SecurityClassification": "PUBLIC"}
-
+  {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "ID": "SearchCriteria ", "Label": "Search Criteria", "FieldType": "SearchCriteria ", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "ID": "hearingRoute ", "Label": "Hearing Route", "FieldType": "Text ", "SecurityClassification": "PUBLIC"}
```
Comment (dattipoeHMCTS) [inline benefit/data/sheets/CaseField/CaseField.json:423]: There's an extra space at the end of some of these fields

File: benefit/data/sheets/CaseTypeTab/CaseTypeTab.json [hunk 3]
```diff
@@ -62,6 +62,7 @@
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "summary_sscs", "TabLabel": "Summary", "TabDisplayOrder": 1, "CaseFieldID": "isScottishCase", "TabFieldDisplayOrder": 54},
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "summary_sscs", "TabLabel": "Summary", "TabDisplayOrder": 1, "CaseFieldID": "reasonableAdjustments", "TabFieldDisplayOrder": 55},
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "summary_sscs", "TabLabel": "Summary", "TabDisplayOrder": 1, "CaseFieldID": "panelDoctorSpecialism", "TabFieldDisplayOrder": 56},
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "summary_sscs", "TabLabel": "Summary", "TabDisplayOrder": 1, "CaseFieldID": "hearingRoute", "TabFieldDisplayOrder": 57},
```
Comment (dattipoeHMCTS) [inline benefit/data/sheets/CaseTypeTab/CaseTypeTab.json:65]: This should be a part of hearing details

**Review by dattipoeHMCTS — CHANGES_REQUESTED** [review]

File: benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json (hunk 1 above)
Comment (MohammedHMCTS) [reply benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json]: Updated now requires full CRUD permissions by all users

File: benefit/data/sheets/CaseTypeTab/CaseTypeTab.json (hunk 3 above)
Comment (MohammedHMCTS) [reply benefit/data/sheets/CaseTypeTab/CaseTypeTab.json]: This has now been moved to listing requirements tab

**Pull request closed without merging**